cypress/e2e/SupplementalApplicationPage/status_update.e2e.js

```diff
@@ -1,25 +1,67 @@
 import { LEASE_UP_LISTING_APPLICATION_ID } from '../../support/consts'
+import { usingFixtures } from '../../support/utils'
 
 const unselectedStatusMenuItem = 'li[aria-selected="false"].dropdown-menu_item > a'
 
-describe('SupplementalApplicationPage status history', () => {
+describe('SupplementalApplicationPage statuses', () => {
+  beforeEach(() => {
+    if (usingFixtures()) {
+      cy.intercept('api/v1/short-form/**', { fixture: 'shortForm.json' }).as('shortForm')
+      cy.intercept('GET', 'api/v1/applications/**/field_update_comments', {
+        fixture: 'fieldUpdateComments.json'
+      }).as('fieldUpdateCommentsGet')
+      cy.intercept('api/v1/applications/**/leases', { fixture: 'leases.json' }).as('leases')
+      cy.intercept('api/v1/rental-assistances?application_id=**', {
+        fixture: 'rentalAssistances.json'
+      }).as('rentalAssistances')
+      cy.intercept('api/v1/supplementals/**', { fixture: 'supplementals.json' }).as('supplementals')
+      cy.intercept('api/v1/supplementals/units?listing_id=**', { fixture: 'units.json' }).as(
+        'units'
+      )
+      cy.intercept('api/v1/lease-ups/listings/**', { fixture: 'leaseUpListing.json' }).as(
+        'leaseUpListing'
+      )
+    } else {
+      cy.intercept('api/v1/short-form/**').as('shortForm')
+      cy.intercept('GET', 'api/v1/applications/**/field_update_comments').as(
+        'fieldUpdateCommentsGet'
+      )
+      cy.intercept('api/v1/applications/**/leases').as('leases')
+      cy.intercept('api/v1/rental-assistances?application_id=**').as('rentalAssistances')
+      cy.intercept('api/v1/supplementals/**').as('supplementals')
+      cy.intercept('api/v1/supplementals/units?listing_id=**').as('units')
+      cy.intercept('api/v1/lease-ups/listings/**').as('leaseUpListing')
+    }
+  })
   it('should allow status updates via the Add a Comment button in the status history section', () => {
     cy.visit('http://localhost:3000/')
     cy.login()
     cy.visit(`/lease-ups/applications/${LEASE_UP_LISTING_APPLICATION_ID}`)
+    cy.wait('@shortForm')
+    cy.wait('@fieldUpdateCommentsGet')
+    cy.wait('@leases')
+    cy.wait('@rentalAssistances')
+    cy.wait('@supplementals')
+    cy.wait('@units')
+    cy.wait('@leaseUpListing')
 
     // Click on the "Add a Comment" button in the Status History sidebar
     cy.get('#add-status-history-comment').click()
 
     cy.testStatusModalUpdate()
   })
-})
 
-describe('SupplementalApplicationPage action buttons', () => {
   it('should allow status updates via the status dropdown in the sidebar', () => {
     cy.visit('http://localhost:3000/')
     cy.login()
     cy.visit(`/lease-ups/applications/${LEASE_UP_LISTING_APPLICATION_ID}`)
+    cy.wait('@shortForm')
+    cy.wait('@fieldUpdateCommentsGet')
+    cy.wait('@leases')
+    cy.wait('@rentalAssistances')
+    cy.wait('@supplementals')
+    cy.wait('@units')
+    cy.wait('@leaseUpListing')
```

Comment:
issue: this test makes a POST request to the `field_update_comments` endpoint, that is not getting intercepted.

Reply:
Thank you for catching this! Looking into it right now

Reply:
I added the missing intercept!